cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Probably should remove this example fixture if we're not using it

Reply:
Removed

Reply:
It looks like this file is still here?

Reply:
No idea what happened there.. It's gone now